### PR TITLE
Add QuicLoad Tool

### PR DESCRIPTION
**Pull request by nibanks**
A tool to test creating multiple H3 connections.

### CODE REVIEW

File: src/tools/load/load.cpp [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+const MsQuicApi* MsQuic;
+volatile long ConnectedCount;
+volatile long ConnectionsActive;
+
+void ResolveServerAddress(const char* ServerName, QUIC_ADDR& ServerAddress) {
+    CxPlatSystemLoad();
+    CxPlatInitialize();
+    CXPLAT_DATAPATH* Datapath = nullptr;
+    //QuicAddrSetFamily(&ServerAddress, AF_INET);
```
Comment (anrossi) [inline src/tools/load/load.cpp:20]: not needed anymore?

File: src/tools/load/load.cpp [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    const char* ServerName = argv[1];
+    QuicAddr ServerAddress = {0};
+    ResolveServerAddress(ServerName, ServerAddress.SockAddr);
+
+    const uint32_t ConnectionCount = argc > 2 ? atoi(argv[2]) : 100;
+    const uint32_t KeepAliveMs = argc > 3 ? atoi(argv[3]) : 60 * 1000;
+    const uint32_t PollMs = argc > 4 ? atoi(argv[4]) : 10 * 1000;
+    const bool ShareUdp = argc > 5 ? (atoi(argv[5]) ? true : false) : true;
+
+    MsQuic = new(std::nothrow) MsQuicApi;
```
Comment (anrossi) [inline src/tools/load/load.cpp:57]: Does this call `MsQuicOpen` and all that? wow!

**Review by anrossi — APPROVED** [review]

File: src/tools/load/load.cpp (hunk 1 above)
Comment (nibanks) [reply src/tools/load/load.cpp:20]: I'm leaving it there for manual usage in the future, but I don't want it in by default.

File: src/tools/load/load.cpp (hunk 2 above)
Comment (nibanks) [reply src/tools/load/load.cpp:57]: Yep. The power of C++!

**Pull request merged**